### PR TITLE
remove: Unused website image assets

### PR DESCRIPTION
**Pull request by Josmithr**
Removes image files that are not referenced by any website documents or infrastructure.

### CODE REVIEW

File: docs/static/images/container-and-component-loading-1.jpg
Comment (alexvy86) [inline docs/static/images/container-and-component-loading-1.jpg:1]: All these `container-and-component-loading-N.jpg` seem to be referenced:
image

**Review by alexvy86 — COMMENTED** [review]
The idea is that none of these were found referenced as `images/<name>` somewhere in docs?

File: docs/static/images/container-and-component-loading-1.jpg
Comment (Josmithr) [reply docs/static/images/container-and-component-loading-1.jpg:1]: Hmm. My search must have been misconfigured. Hopefully this fails the build. I'll do a second pass.

Comment (Josmithr) [reply docs/static/images/container-and-component-loading-1.jpg:1]: It didn't break the build... that is very unfortunate...

**Comment by jikim-msft** [comment]
https://github.com/Josmithr/FluidFramework/blob/docs/remove-unused-images/docs/content/docs/data-structures/tree.md#schema-definition

I also found that the image is broken for `A screenshot of a sticky note board app`.

Screenshot 2024-10-09 at 12 19 54

Chaning from `images/sticky-note-board-app.png` to `/docs/static/images/sticky-note-board-app.png` fixes the issue. And I think we can remove the `images/sticky-note-board-app.png`.

File: docs/static/images/container-and-component-loading-1.jpg
Comment (jikim-msft) [reply docs/static/images/container-and-component-loading-1.jpg:1]: @Josmithr 

Can we create a follow-up item on this? Maybe we can break the build, if the image that the `.md` references to does not exist.

Comment (Josmithr) [reply docs/static/images/container-and-component-loading-1.jpg:1]: Okay. I figured out why it didn't fail. Those images are referenced by a `draft` document. Those documents aren't built by default, so these were missed. I'm running a local build with draft docs included to give a better audit.

Longer term we should probably do a draft build in the pipeline as a sanity check.

Comment (Josmithr) [reply docs/static/images/container-and-component-loading-1.jpg:1]: I was incorrect. Image references don't seem to get checked at all, as far as I can tell. This is a serious problem. Fortunately, I've confirmed that Docusaurus's _does_ fail for missing images.

I'm going to go ahead and close this PR - we can clean up images once we've migrated to the new infra.

**Pull request closed without merging**